packages/wonder-stuff-testing/src/jest/__tests__/isolate-modules.test.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+import {isolateModules} from "../isolate-modules.js";
+
+describe("#isolateModules", () => {
```

Comment:
I added some tests - just because we can't mock everything doesn't mean we can't test it. :)